allensdk/brain_observatory/ecephys/write_nwb/_schemas.py

```diff
@@ -27,7 +27,11 @@ class Channel(RaisingSchema):
     probe_horizontal_position = Int(required=True)
     manual_structure_id = Int(required=True, allow_none=True)
     manual_structure_acronym = String(required=True, allow_none=True)
-
+    cortical_layer = String(required=True, allow_none=True)
+    AP_coordinate = Float(required=True, allow_none =True)
```

Comment:
these are not tracked in lims

Reply:
What do you want to do with the write_nwb changes Josh made? I thought you said Josh was using these changes to generate the spike files to run against his most recent changes to EcephysSession and the StimAnalysis classes. I'll roll back the changes to write_nwb but I don't know if/how it's going to break the ecephys code.